### PR TITLE
Update RedOS 7.3

### PR DESCRIPTION
**Pull request by foxzi**
Update RedOS from version 7.3.1 to 7.3.3. Also, add new image based on RedOS image from official registry. More information about RedOS image https://redos.red-soft.ru/base/arm/arm-other/docker-install/. RedOS now uses dnf instead of yum.

### CODE REVIEW

**Review by Totktonada — APPROVED** [review]
No objections from my side.

File: redos/7.3/Dockerfile [hunk 1]
```diff
@@ -1,103 +1,32 @@
-FROM centos:7 as prepare
+FROM registry.red-soft.ru/ubi7/ubi-minimal:latest
```
Comment (ylobankov) [inline redos/7.3/Dockerfile]: In the commit message you state that it is 7.3.3 version, but the tag is `latest`. It's very strange and absolutely wrong.

```console
curl -X GET https://registry.red-soft.ru/v2/ubi7/ubi-minimal/tags/list | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   369  100   369    0     0   3723      0 --:--:-- --:--:-- --:--:--  3727
{
  "name": "ubi7/ubi-minimal",
  "tags": [
    "7.3.2-230321",
    "7.3.3-230925",
    "7.3.2-230323",
    "7.3.2-230809",
    "7.3.2-230515",
    "7.3.3-230911",
    "7.3.2-230503",
    "7.3.2-230821",
    "7.3.2-221227",
    "7.3.2-230419",
    "7.3.2-230814",
    "7.3.2",
    "7.3.2-230731",
    "7.3.2-230411",
    "7.3.2-230712",
    "7.3.3-230904",
    "7.3.3-231002",
    "latest",
    "7.3.3-230918",
    "7.3.3-231009",
    "7.3.2-230608",
    "7.3.3-230926",
    "7.3.3-230828"
  ]
}
```

As you can see, `7.3.3-231009` is the latest 7.3.3 tag. Please use it.

File: redos/7.3/Dockerfile [hunk 2]
```diff
@@ -1,103 +1,32 @@
...
-    yum \
-    rpm-devel \
-    rpmdevtools \
-    rpm-build \
-    ccache \
-    git \
-    sudo \
-    dnf-utils
+RUN dnf -y update && \
+        dnf -y install \
```
Comment (ylobankov) [inline redos/7.3/Dockerfile]: Please align this and all following lines with the line above. It should be one column.

Comment (foxzi) [reply redos/7.3/Dockerfile]: fixed

File: redos/7.3/Dockerfile (hunk 1 above)
Comment (foxzi) [reply redos/7.3/Dockerfile]: Fixed

**Review by ylobankov — APPROVED** [review]

**Pull request merged**